charts/pravega/README.md

```diff
@@ -13,7 +13,7 @@ This chart creates a [Pravega](https://github.com/pravega/pravega) cluster in [K
   - An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator)
   - An existing Apache Bookkeeper 4.9.2 cluster. This can be easily deployed using our [BookKeeper Operator](https://github.com/pravega/bookkeeper-operator)
   - Pravega Operator. You can install it using its own [Helm chart](https://github.com/pravega/pravega-operator/tree/master/charts/pravega-operator)
```

Comment:
Here we could just say, for Deploying the operator please refer 'this' and just link to file charts/pravega-operator/Reademe.md
There is no need to make it explicit that we do this using helm charts.

Reply:
Done